### PR TITLE
MINOR: server: adds slowstart parameter

### PR DESCRIPTION
**Pull request by schegi**
Adds support for slowstart parameter to server configuration part of the dataplaneapi specification.
Related: https://github.com/haproxytech/models/pull/16

### CODE REVIEW

**Comment by mjuraga** [comment]
Any reason for closing this?

**Comment by Mo3m3n** [comment]
@schegi 
This is the right place for your patch.
I see that you closed this PR and opened it in HAProxy ML https://www.mail-archive.com/haproxy@formilux.org/msg37212.html
HAProxy ML is used mainly for the HAProxy program

**Comment by schegi** [comment]
Nothing was happening for a couple of days, the i reread the CONTRIBUTING guide and there was mentioned to not open PRs but to post patches on the mailing list, i thought it was just ignored cause of wrong channel.
Still have to PRs pending, on on native-client and one for dataplane api. Waiting for those to get merged. Also reopend the one on models.

File: models/configuration.yaml [hunk 1]
```diff
@@ -798,6 +798,9 @@ server:
     send-proxy-v2:
       type: string
       enum: [enabled, disabled]
+    slowstart:
+      type: string
+      pattern: '^[1-9][0-9]*(us|ms|s|m|h|d)$'
```
Comment (mjuraga) [inline models/configuration.yaml]: This is correct given the docs, but in the API we handle all time values as `ms` which is default for haproxy, it's just easier to work in it with API, so when we read string values with suffix from, we report it as equivalent `integer` value in ms in API responses. When writing it back to the config we always write it as integer in ms.

So here `slowstart` should be `type: integer`. Also can you add this parameter to the `default_server` key in `defaults` and `backed` object to be consistent.

**Comment by mjuraga** [comment]
Also sorry for the delay in the review. This is completely on me and my schedule.

**Comment by schegi** [comment]
Thx, added the slowstart parameter to the default_server key. Hope this did the job.

**Comment by mjuraga** [comment]
Tnx, merged now, you can now work on client-native implementation, new models commit will be auto generated from this.

**Pull request merged**